styles/form/form.less

```diff
@@ -135,6 +135,10 @@
 			margin-right: 0.5em;
 		}
 
+		p {
+			display: inline-block;
+		}
```

Comment:
Do you know where the `<p>` comes from? I did a scan for `checkbox_and_radiobutton` and couldn't find anything where a `<p>` was created...

Reply:
Not sure, but it appears in the custom review form

Reply:
This is working I think, something you want me to do about this?